app/pb/api.proto

```diff
@@ -261,6 +275,13 @@ message RepeatedStringValue {
   repeated string value = 2;
 }
 
+message RepeatedLanguageAbilityValue {
+  // A RepeatedLanguageAbilityValue is like a "protobuf wrapper" (like {Type}Value) but for repeated LanguageAbility
+  // allows us to determine if the user wanted to remote this list of LanguageAbility's
+  bool exists = 1;
+  repeated LanguageAbility value = 2;
```

Comment:
Will the new list sent in the API completely replace my existing list, or will only the mentioned languages be updated? Please write a comment about that.